test-apis.js

```diff
@@ -9,14 +9,20 @@ const IPFS = require('ipfs')
 let jsIpfs = {
   'js-ipfs': {
     type: 'proc',
-    exec: IPFS
+    test: true,
+    disposable: true,
+    ipfsModule: require('ipfs')
   }
 }
 
 const goIpfs = {
   'go-ipfs': {
     type: 'go',
-    IpfsClient: require('ipfs-http-client')
+    test: true,
+    disposable: true,
+    args: ['--enable-pubsub-experiment'],
+    ipfsHttpModule: require('ipfs-http-client'),
+    ipfsBin: require('go-ipfs-dep').path()
```

Comment:
Something is still wrong with go-ipfs where it doesn't stop. Wondering if it requires a proper `factory.clean`, even though all `factory.clean` does is stop all its controllers... could still be holding on to some reference though.

Reply:
Skipped the tests for now :/